AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/config/NewConfig.java

```diff
@@ -156,6 +158,8 @@ public void loadDefaults() {
         addDefault("per-command-warm-ups.home", "default", "Warm-up timer for /home");
         addDefault("per-command-warm-ups.back", "default", "Warm-up timer for /back");
 
+        addDefault("blindness-on-warmup", false, "Adds a blindness effect while countdown is playing.");
+
```

Comment:
Could be worded slightly better - let's say "Gives the teleporting player a blindness effect whilst waiting to teleport."